west.yml

```diff
@@ -93,7 +93,7 @@ manifest:
       groups:
         - hal
     - name: hal_nxp
-      revision: 9512229c35e230003c64fda41be578f2718b70f4
+      revision: 90db601fa84d203fd6d731cba0e391016b66e319
```

Comment:
Check this commit hash value. This is causing CI failures

Reply:
I also noticed the west setup failing on updating nxp hal earlier and I double checked this commit hash twice now, I don't know what the problem is yet though